frontend/src/layout/navigation/SideBar/SideBar.scss

```diff
@@ -60,6 +60,12 @@
     }
 }
 
+.Sidebar__buttons {
+    > * + * {
```

Comment:
While it won't cause big issues, these kinds of selectors don't perform very well. Specifically the wildcards. The combinator approach is super good, but if we can reference a common class or element name that selector would be much faster.

Reply:
Is there something you can point me to that indicates a performance problem? 

We use this a lot and it's an awesome tool for having reliable repeatable layout separation. The benefit here is we don't need to consider the content and how it changes (if the top level if an anchor tag or a button it will still receive the same treatment) 💪

Reply:
Swapped over to li in this case but I'm interested if you have a source for the performance issues as we do still use this pattern a lot elsewhere...

Reply:
Universal element selectors have to parse every element in the hierarchy to figure out matches. It's not a huge deal with the latest matching, but it does make that declaration have to scan a really broad area of elements. If we could use a class or a common parent element selector, it'd limit the scope and probably reduce the chance of unintended matches. Your call of course, I just think it could be more specific to the components we're trying to style and would be more robust to future changes in that navigation component.